x/farming/keeper/proposal_handler.go

```diff
@@ -181,7 +181,16 @@ func (k Keeper) DeletePublicPlanProposal(ctx sdk.Context, proposals []types.Dele
 			return err
 		}
 
-		k.RemovePlan(ctx, plan)
+		// Send all remaining coins in the farming pool to the termination address.
+		if plan.GetFarmingPoolAddress().String() != plan.GetTerminationAddress().String() {
+			balances := k.bankKeeper.GetAllBalances(ctx, plan.GetFarmingPoolAddress())
```

Comment:
I think we need to consider using `SpendableCoins` instead of all `GetAllBalances` before `SendCoins`, `InputOutputCoins` to prevent insufficient errors occurring when the account has a locked amount (vesting)